katcp/inspecting_client.py

```diff
@@ -366,7 +370,8 @@ def connect(self, timeout=None):
         try:
             yield maybe_timeout(self.katcp_client.until_running())
             self._logger.debug('Katcp client running')
-        except tornado.gen.TimeoutError:
+        except Exception as err:
+            self._logger.error("Failed to start katcp_client: %s", str(err))
```

Comment:
`str` is redundant with `%s`
```suggestion
            self._logger.error("Failed to start katcp_client: %s", err)
```